assets/css/index.css

```diff
@@ -285,4 +289,31 @@ footer {
 
 .code-text {
   font-family: monospace;
-}
\ No newline at end of file
+}
+.code-block-container {
+  position: relative;
+}
+button.clipboard {
+  color: var(--main-color);
+  border-radius: 4px;
+  border: transparent;
+  margin: 5px;
+  justify-items: right;
+  background-color: transparent;
+  cursor: pointer;
+}
+button.clipboard:hover {
+  color: white;
+  background-color: var(--main-color);
+}
+button.clipboard:after {
+  content: "\f328";
+  font-size: 15px;
+  vertical-align: bottom;
+  font-family: "Font Awesome 5 Free";
+}
+.div-copy {
+  text-align: right;
```

Comment:
in case I use that, it overlaps with the markup
![image](https://user-images.githubusercontent.com/59360600/92766798-2f36ba00-f3b4-11ea-9b0f-a635d3a5ea0e.png)

Reply:
Good point. Then adjust it to the following:
1. for `.div-copy`:
```
position: absolute;
top: 9px;
right: 0;
background-color: #f5f2f0;
```
2. for `.language-markup`:
```
padding-right: 2em;
```